.travis.yml

```diff
@@ -11,37 +11,11 @@ env:
     secure: MW6tNPPUcb7rxTXb1XvOQA6VQfwZSKjbnlNuTj/z+WPGzas5tjUShLfF0BcknOP8IVyYWV14jTKn89SDhlZi3oYw8xVJxbmH049c4qab5/SMTV0wFjl85Kgmt1aIUTdtDHBNrfYDHpGUr80nuLFMq+kgkhflDOU50p22fZ0gtiNftFn6mvnqkoC9rZBUis0SXG3NqEDjdJKYtNRsjlN9UX4oKHW4qwiOrJULNbfJVSkSx9Wq/ZfIRAb4YOIQt5lL89ory5I6dnyMje32cikJH7jJ6JhYckvYrULLsvl7zgmjpw8zncKy95+HYg2huK84TvvhfajWzGbb1oIUFfqdd0bjxnD6AJlQ/Ybbs7RhSyJ/3306I0zXoOqr0EHVhVS4iy9FZOaBQkwykOhmZVs67c+9J0ONNT7eiX8wFue9NYLVFHNq7afOfggfuXv250j78tFqSH+ytomKYNzrvchbLa7VsC0IMtesYg6oTlEpik51FTJmuE8UjpzC27jAGhOiprmUSl0Zfnb+sd5UsTEBLLDv79V2CWHU2WCXksrLp597fkDUPJTBYCDv4OojDf7vwyR5GOwwDCmsCufx7bLVw8Wr4TXztw96Wshxq2TDDf4L43DreFx3frt8vkWqbHFHI0/1ssO8MBGn0/kRndZyXAtIgmQxtY7DRcdo0UcUw7k=
```

Comment:
I would remove these encrypted passwords that I think were used only in the manylinux build?